modules/gltf/editor/editor_scene_importer_blend.cpp

```diff
@@ -467,7 +455,7 @@ bool EditorFileSystemImportFormatSupportQueryBlend::query() {
 		EditorNode::get_singleton()->get_gui_base()->add_child(browse_dialog);
 	}
 
-	String path = EDITOR_GET("filesystem/import/blender/blender3_path");
+	String path = EDITOR_GET("filesystem/import/blender/blender_path");
 
 	if (path == "") {
```

Comment:
Removal of `blender.exe`/`blender` executable name seems reasonable, but it will break all autodetection logic here (which will never include executable name), so it should be updated as well.

Reply:
I'm not sure what do you mean by this, could you elaborate a bit?

This change is to make the blender_path require pointing to the actual binary instead of a directory (Since this was causing issues with picking a specific blender version)

Reply:
I think I know what you mean now, you mean the behavior for when blender_path is set to "" as in the user did not set a path.

Reply:
> I'm not sure what do you mean by this, could you elaborate a bit?

There's autodetect logic if path is empty, e.g, few lines below it's trying to check `/opt/homebrew/bin`, but since `blender` name in no longer appended, it should be changed to `/opt/homebrew/bin/blender` to find it (similar for the other cases).

Reply:
Yeah, I'm updating that logic now as well.

Reply:
I'm not familiar with the OSX .app location of blender, would it make sense to have it check: `/Applications/Blender.app/Contents/MacOS/blender` ? or `/Applications/Blender.app/Contents/MacOS/Blender` ?